webapp/channels/src/components/login/login.tsx

```diff
@@ -145,6 +149,12 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
     const query = new URLSearchParams(search);
     const redirectTo = query.get('redirect_to');
 
+    const [desktopToken, setDesktopToken] = useState('');
+    const [desktopAuthLogin, setDesktopAuthLogin] = useState(query.get('desktopAuthStatus') === 'complete' ? DesktopAuthStatus.Complete : DesktopAuthStatus.None);
+    const desktopAuthInterval = useRef<number>();
```

Comment:
I think the type of the ref value should actually be `NodeJS.Timer` to avoid needing type assertions

Reply:
You end up needing to assert it in a different place now as `clearInterval` expects a `number`. Though I'd say `NodeJS.Timer` is more accurate, so I'll use it.